addons/map_gestures/functions/fnc_getProximityPlayers.sqf

```diff
@@ -0,0 +1,23 @@
+/*
+ * Author: Dslyecxi, MikeMatrix
+ * Returns all players in a given range and in the units vehicle.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Range <NUMBER>
+ *
+ * Return Value:
+ * All units in proximity <ARRAY>
+ *
+ * Example:
+ * ["example value"] call ace_module_fnc_functionName
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_unit", "_range"];
+
+_proximityPlayers = (getPos _unit) nearEntities [["CAMAnBase"], _range];
+_proximityPlayers = _proximityPlayers - [_unit];
```

Comment:
It might be better to use this instead:
```sqf
_proximityPlayers deleteAt (_proximityPlayers find _unit);
```
Though I suspect that `_unit` is always the first element on the entities array, in which case you won't need a find.